### ISSUE TITLE
[Effect Suggestion] My Super Cool Effect Idea

### ISSUE THREAD

**Issue by dumbdumb89** [gta-chaos-mod/ChaosModV#903]
**Name of effect**
COVID-19

**Duration if timed effect (long or short timed?)**
Short (30s)

**Effect Description**
All peds have a surgical mask on their face, constantly cough, and you die if you go within 6ft/2m of anyone

**Comment by Thunderstorm64** [comment]
You can't add props to NPC's that don't already exist as far as I'm aware

**Comment by dumbdumb89** [comment]
> You can't add props to NPC's that don't already exist as far as I'm aware

Hmm. I think the coroners in Dead Man Walking wear surgical masks. I was wondering if it was possible to get the model for those and somehow add them to peds?

**Comment by lZevex** [comment]
> Hmm. I think the coroners in Dead Man Walking wear surgical masks. I was wondering if it was possible to get the model for those and somehow add them to peds?
No, because the masks are merged onto their character, meaning that modified peds would have to be spawned

**Comment by TheRealAyCe** [comment]
"Wear a mask"
Forces the player character to wear a... mask (one of the heist ones)

**Comment by dumbdumb89** [comment]
Removed the mask idea since I now know it's impossible to implement
